Jint/Native/Array/ArrayOperations.cs

```diff
@@ -68,6 +70,60 @@ public virtual JsValue[] GetAll(Types elementTypes)
 
         public abstract void DeletePropertyOrThrow(ulong index);
 
+        public IEnumerator<JsValue> GetEnumerator()
+        {
+            return new ArrayOperationsIterator(this);
+        }
+
+        IEnumerator IEnumerable.GetEnumerator()
+        {
+            return GetEnumerator();
+        }
+
+        private sealed class ArrayOperationsIterator : IEnumerator<JsValue>
+        {
+            private readonly ArrayOperations _obj;
+            private ulong _current;
+            private bool _first;
```

Comment:
_first not really needed?

Reply:
_current is ulong, and after the first `MoveNext()` it needs to be `0`. Because we can't use -1 I decided to use a bool instead of using a bigger signed struct.

Reply:
OK